docs/aegis/roadmap/core-tooling-governance-audit.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Core Tooling Governance Audit
+area: Governance
+status: Planning
+funding_required: false
+# TODO: Can we disclose the sponsor?
+# sponsors: ...
+index: 5
+previous:
+  url: core-tooling-compliance
+  title: Core Tooling Compliance
+next:
+  url: cna
+  title: Erlang Ecosystem Foundation CNA
+---
+
+## Goal
+
+Define Governance structure for core projects
+
+## Impact
+
+By conducting a thorough governance audit and establishing a clear governance
+structure for pivotal open source projects (Erlang, Elixir, Gleam, and Hex),
+this milestone ensures transparency, continuity, and trust across the ecosystem.
+Deliverables include a detailed checklist to guide governance best practices,
+the formal documentation of existing structures, and explicit contingency plans
+to address maintainer incapacitation. Additionally, efforts to encourage
+Ericsson’s formal governance process for Erlang, along with guidance for other
+projects to publish their governance documentation, underscore the commitment to
+robust oversight. Finally, by codifying rules around IP, copyright, patents, and
```

Comment:
Regarding IP, copyright, and patents, aren't licenses already specifying who owns the copyright (IP) and the license places restrictions on how the Work can be exploited/used/distribute?

I guess I am not sure about what is implied with the "codification" of IP, copyright, and patens.

Reply:
Sorry, I misread. I'll reword it.